src/Pages/Signup/Component/SignupInput.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export default function SignupInput({ theme, title, subTitle, guideText }) {
+  return (
+    <InputWrapper>
+      <div>{title}</div>
+      {subTitle !== undefined && <div>{subTitle}</div>}
```

Comment:
```suggestion
      {subTitle && <div>{subTitle}</div>}
```
true일 경우만 렌더링하면 된다면 이렇게는 어떨까요?

Reply:
아~ 몰랐네요 감사합니다